roles/node_exporter/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# defaults file for node_exporter
```

Comment:
empty file is empty. also, it doesn't end in a newline, which our editorconfig says it should